config/locales/en.yml

```
@@ -328,6 +330,7 @@ en:
       created_by_unknown: Unknown
       name: Group name
     index:
+      active_empty_message: You're not in any active groups.
```

Very minor but we try to use curly quotes in translations:
```suggestion
      active_empty_message: You’re not in any active groups.
```

---

Thanks! Updated